lib/msal-browser/src/config/Configuration.ts

```diff
@@ -66,6 +67,7 @@ export type BrowserSystemOptions = SystemOptions & {
     windowHashTimeout?: number;
     iframeHashTimeout?: number;
     loadFrameTimeout?: number;
```

Comment:
Should we add backwards compatibility/deprecation log messages for existing users using this parameter?

Reply:
Nvm, backwards compat isn't an option since it is used as defaults for the hashTimeouts. However I think a deprecation/change-of-behavior log message would be good.